pkgs/applications/misc/octoprint/default.nix

```diff
@@ -16,6 +17,30 @@ let
     self = py;
     packageOverrides = lib.foldr lib.composeExtensions (self: super: { }) (
       [
+        (
+          # Due to flask > 2.3 the login will not work
```

Comment:
Please create/find a relevant issue upstream and link to it.

Reply:
`octoprint` pins their version of `flask` and `werkzeug`, therefore there is no upstream issue. I still created an heads-up issue upstream so the developers can be aware of the upcoming issue..